src/signals/incident-management/containers/IncidentOverviewPage/contants.ts

```diff
@@ -20,23 +20,25 @@ export enum SortOptions {
 }
 
 export enum SortOptionKeys {
-  ADRES = 'adres',
-  DATUM = 'datum',
+  ADDRESS = 'adres',
+  DATE = 'datum',
   ID = 'id',
-  STADSDEEL = 'stadsdeel',
+  DISTRICT = 'stadsdeel',
   STATUS = 'status',
   SUBCATEGORY = 'subcategorie',
-  URGENTIE = 'urgentie',
+  PRIORITY = 'urgentie',
+  ASSIGNED_USER_EMAIL = 'toegewezen_aan',
```

Comment:
This key is used in the sorting description, so it shows `Sorteer op toegewezen_aan (a-z)` now. Shouldn't we use the label instead of the key for that? And turn it into lowercase using CSS?

If we do, I think we can get rid of the `key` property in `sortOptionsList` entirely?

Reply:
Sharp